iDiff/utils/image_prep_utils.go

```diff
@@ -157,11 +133,39 @@ func (p CloudPrepper) ImageToFS() (string, error) {
 	return getImageFromTar(tarPath)
 }
 
+func (p CloudPrepper) getConfig() (ConfigSchema, error) {
+	ref, err := docker.ParseReference("//" + p.Source)
```

Comment:
It's just what Dan discovered was necessary to make it work... @dlorenc I can't remember if you said you knew the reason?